tpm2/test/tpm2_test.go

```diff
@@ -1478,3 +1478,46 @@ func TestPlainImport(t *testing.T) {
 		t.Errorf("Got %X, expected %X", outBuff, inBuff)
 	}
 }
+
+func TestPolicyPCR(t *testing.T) {
```

Comment:
This is a great test. We should also add in a test for passing `nil` for the expected digest.

Reply:
Done